### PR TITLE
Added Pytest xfail marker for FOGL-7712

### PR DESCRIPTION
**Pull request by Mohit04tomar**
I ran the following job on the latest commit(530d9ed15f401154006add2df688cf9ef8631120) of this pullrequest:
http://jenkins.dianomic.com:8080/job/fledge_packages_system_test_data_availability_notification_rule/163/

### CODE REVIEW

File: tests/system/python/packages/test_rule_data_availability.py [hunk 1]
```diff
@@ -197,6 +197,8 @@ def test_data_availability_single_audit(self, fledge_url, skip_verify_north_inte
         resp2 = utils.get_request(fledge_url, get_url)
         assert len(resp2['audit']) - len(resp1['audit']) == 1, "ERROR: NTFSN not triggered properly on CONCH"
 
+    # Remove pytest skip marker when the FOGL-7712 is fixed
```
Comment (praveen-garg) [inline tests/system/python/packages/test_rule_data_availability.py:200]: This comment should be in Jira 7712 with reference of this PR and CI job.

File: tests/system/python/packages/test_rule_data_availability.py [hunk 2]
```diff
@@ -197,6 +197,8 @@ def test_data_availability_single_audit(self, fledge_url, skip_verify_north_inte
...
+    @pytest.mark.skip(reason="Skipping this test due to FOGL-7712")
```
Comment (praveen-garg) [inline tests/system/python/packages/test_rule_data_availability.py:201]: reason=FOGL-7712

**Review by praveen-garg — CHANGES_REQUESTED** [review]

**Comment by ashish-jabble** [comment]
Skipping tests due to known bugs is generally not a good practice as it can lead to incomplete testing and unresolved issues. Instead, consider marking tests with relevant tags, documenting bugs, and ensuring that there is a plan to fix them. 
Better approach to use with `@pytest.mark.xfail(reason="FOGL-7712")`

**Comment by praveen-garg** [comment]
Expected failure is a better idea.

Though skipping this is also fine as fixing that Jira is not planned and may never happen.

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**